src/admin/pages/Project/index.tsx

```diff
@@ -32,14 +32,10 @@ const ProjectImpl: React.FC = () => {
     resolver: yupResolver(updateProjectSettingSchema()),
   });
 
-  useEffect(() => {
-    if (updatedProjectSetting === undefined) return;
-    enqueueSnackbar(t('toast.updated_successfully'), { variant: 'success' });
-  }, [updatedProjectSetting]);
-
   const onSubmit: SubmitHandler<FormValues> = async (form: FormValues) => {
     try {
       await trigger(form);
+      enqueueSnackbar(t('toast.updated_successfully'), { variant: 'success' });
```

Comment:
If useEffect, it will not trigger unless the value changes. Therefore, display as snackbar after submit.